Sources/BugsnagPerformance/Public/BugsnagPerformanceConfiguration.mm

```diff
@@ -58,6 +58,7 @@ + (instancetype)loadConfigWithInfoDictionary:(NSDictionary * _Nullable)infoDicti
     auto autoInstrumentAppStarts = BSGDynamicCast<NSNumber>(bugsnagPerformanceConfiguration[@"autoInstrumentAppStarts"]);
     auto autoInstrumentViewControllers = BSGDynamicCast<NSNumber>(bugsnagPerformanceConfiguration[@"autoInstrumentViewControllers"]);
     auto autoInstrumentNetworkRequests = BSGDynamicCast<NSNumber>(bugsnagPerformanceConfiguration[@"autoInstrumentNetworkRequests"]);
+    auto samplingProbability = BSGDynamicCast<NSNumber>(bugsnagPerformanceConfiguration[@"samplingProbability"]);
```

Comment:
Can we call it `overriddenSamplingProperty`? Otherwise it will start to get confusing trying to remember what's what ;-)

Reply:
As discussed IRL, we'll keep it as `samplingProbability` as I think it makes sense from a developer's perspective and the name has already started being released elsewhere.